crates/dips/proto/gateway.proto

```diff
@@ -2,7 +2,7 @@ syntax = "proto3";
 
 package graphprotocol.gateway.dips;
 
-service DipsService {
```

Comment:
I would suggest naming it `GatewayDipsService.`

I want to rename the "Dipper" to DIPs Gateway at some point in the future.